mirror_csi_images/scripts/image-mirror.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+set -uo pipefail
+export DOCKER_CLI_EXPERIMENTAL="enabled"
+
+ARCH_LIST="amd64 arm64 arm s390x"
```

Comment:
Does CSI upstream support `arm` as well?

Reply:
it varies from different packages and different versions (tags).
the complete CSI registry can be found here: https://console.cloud.google.com/gcr/images/k8s-staging-sig-storage

Reply:
Sounds good.